<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ISOBrowserWindow</class>
 <widget class="QWidget" name="ISOBrowserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISO Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/duckstation-qt.qrc">
    <normaloff>:/icons/media-optical.png</normaloff>:/icons/media-optical.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="openPath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="openFile">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="directoryView">
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
     </widget>
     <widget class="QTreeWidget" name="fileView">
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="extract">
       <property name="text">
        <string>Extract</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extractMode2">
       <property name="text">
        <string>Extract (XA)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extractRaw">
       <property name="text">
        <string>Extract (Raw)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/duckstation-qt.qrc"/>
 </resources>
 <connections/>
</ui>
